### ISSUE TITLE
After setting up the git-commits plugin, the user avatar displayed at the bottom of the article cannot be loaded, and the jump link is wrong.

### ISSUE THREAD

**Issue by qianmoQ** [squidfunk/mkdocs-material#4176]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [ ] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [ ] ... the documentation does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem

### Description

After setting up the git-commits plugin, the user avatar displayed at the bottom of the article cannot be loaded, and the jump link is wrong.

### Expected behaviour

Show the correct avatar and jump link

### Actual behaviour

None

### Steps to reproduce

go to https://opensource.edurt.io/projects.html

### Package versions

- Python: `3.9`
- MkDocs: `1.3.0`
- Material: `8.3.9+insiders.4.21.0`

### Configuration

```yaml
site_name: OpenSource ｜ EdurtIO
site_url: https://opensource.edurt.io/
site_author: qinamoQ
site_description: >-
  Committed to building friendly open source projects

repo_name: EdurtIO/opensource.edurt.io
repo_url: https://github.com/EdurtIO/opensource.edurt.io
edit_uri: "https://github.com/EdurtIO/opensource.edurt.io/docs"

copyright: Copyright &copy; 2022 EdurtIO

theme:
  name: material
  custom_dir: overrides
  icon:
    logo: material/open-source-initiative
  favicon: material/open-source-initiative

  static_templates:
    - 404.html

  include_search_page: false
  search_index_only: true

  language: en
  features:
    - content.code.annotate
    - content.tabs.link
    - navigation.expand
    - navigation.indexes
    - navigation.sections
    - navigation.tabs
    - navigation.top
    - navigation.tracking
    - search.highlight
    - search.share
    - search.suggest
    - content.tooltips
  palette:
    - scheme: default
      primary: indigo
      accent: indigo
      toggle:
        icon: material/toggle-switch-off-outline
        name: Switch to dark mode
    - scheme: slate
      primary: red
      accent: red
      toggle:
        icon: material/toggle-switch
        name: Switch to light mode
  font:
    text: Roboto
    code: Roboto Mono

extra:
  social:
    - icon: fontawesome/brands/github
      link: https://github.com/EdurtIO
    - icon: fontawesome/brands/linkedin
      link: https://linkedin.com/in/qianmoQ/

  consent:
    title: Cookie consent
    description: >- 
      We use cookies to recognize your repeated visits and preferences, as well
      as to measure the effectiveness of our documentation and whether users
      find what they're searching for. With your consent, you're helping us to
      make our documentation better.

markdown_extensions:
  - admonition
  - abbr
  - attr_list
  - def_list
  - footnotes
  - meta
  - md_in_html
  - tables
  - toc:
      permalink: true
  - pymdownx.arithmatex:
      generic: true
  - pymdownx.betterem:
      smart_enable: all
  - pymdownx.caret
  - pymdownx.details
  - pymdownx.emoji:
      emoji_index: !!python/name:materialx.emoji.twemoji
      emoji_generator: !!python/name:materialx.emoji.to_svg
  - pymdownx.highlight
  - pymdownx.inlinehilite
  - pymdownx.keys
  - pymdownx.snippets
  - pymdownx.magiclink:
      repo_url_shorthand: true
      user: squidfunk
      repo: mkdocs-material
  - pymdownx.mark
  - pymdownx.smartsymbols
  - pymdownx.superfences
  - pymdownx.tabbed:
      alternate_style: true 
  - pymdownx.tasklist:
      custom_checkbox: true
  - pymdownx.tilde
  - pymdownx.emoji:
      emoji_index: !!python/name:materialx.emoji.twemoji
      emoji_generator: !!python/name:materialx.emoji.to_svg

use_directory_urls: false

extra_css:
  - stylesheets/extra.css

plugins:
  - search

  - tags:
      tags_file: tags.md

  - i18n:
      default_language: en
      languages:
        en: English
      nav_translations:
        en:
          project_index: Project Index
          project_dbm: DBM (DataBase GUI)

  - git-revision-date-localized:
      enable_creation_date: true
      fallback_to_build_date: true

  - git-committers:
      repository: EdurtIO/opensource.edurt.io

  - git-authors

  - redirects:
      redirect_maps:
        publish-project-new.md: project/dbm.md

nav:
  - Home: index.md
  - Projects: 
      - project_index: projects.md
      - project_dbm: project/dbm.md
```

### System information

- GitHub Actions

**Comment by squidfunk** [comment]
Thanks for reporting. This appears to be a bug related to the usage of a gravatr URL as opposed to an image managed by GitHub. The URL is invalid:

```
https://www.gravatar.com/avatar/d09d858a272052ccc6af0710eb2cb869?d=identicon?size=72
```

This is a bug which I'll look into asap.

**Comment by qianmoQ** [comment]
> Thanks for reporting. This appears to be a bug related to the usage of a gravatr URL as opposed to an image managed by GitHub. The URL is invalid:
> 
> ```
> https://www.gravatar.com/avatar/d09d858a272052ccc6af0710eb2cb869?d=identicon?size=72
> ```
> 
> This is a bug which I'll look into asap.

Is there a custom configuration here to extract the avatar address of github?

**Comment by squidfunk** [comment]
The avatar address is extracted by the git-committers plugin, so maybe ask the maintainers.

**Comment by squidfunk** [comment]
Fixed in squidfunk/mkdocs-material-insiders@f4b5c4da9. The URL is now checked for existing query parameters, and if there are any, a `&` is used.

**Comment by squidfunk** [comment]
Released as part of 8.4.0+insiders-4.21.1.
